cleverhans/discretization/Discretization Example.ipynb

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
I think this should go in cleverhans/examples rather than in a new root-level directory